<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngMenuPopup</class>
 <widget class="QWidget" name="EngMenuPopup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>389</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>389</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>375</width>
     <height>389</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>389</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/graphics/empopup/bg.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgTitle">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>8</y>
      <width>361</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;</string>
    </property>
    <property name="text">
     <string>Engineering Menu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbnucleus">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>75</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
color: rgb(0, 85, 0);
}
*:disabled{
 color:#547508;
}

</string>
    </property>
    <property name="text">
     <string>Nucleus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>322</y>
      <width>99</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
        min-width: 99px;
        max-width: 99px;
        min-height: 49px;
        max-height: 49px;
        border: 0px;
        color: #ffffff;
}

*:pressed {
        background-image: url(:/graphics/empopup/buttononPress.png);
}

</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbseperator_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>180</y>
      <width>359</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/graphics/empopup/img_divider.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>322</y>
      <width>99</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
        min-width: 99px;
        max-width: 99px;
        min-height: 49px;
        max-height: 49px;
        border: 0px;
        color: #ffffff;
}

*:pressed {
        background-image: url(:/graphics/empopup/buttononPress.png);
}

</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Scrolling">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>128</y>
      <width>143</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
        min-width: 143px;
        max-width: 143px;
        min-height: 48px;
        max-height: 48px;
        border: 0px;
        background-image: url(:/graphics/empopup/sliderOff.png);	
}


*:checked
{ 
 background-image: url(:/graphics/empopup/sliderOn.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbseperator">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>121</y>
      <width>359</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="graphics.qrc">:/graphics/empopup/img_divider.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Nucleus">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>67</y>
      <width>143</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	
        min-width: 143px;
        max-width: 143px;
        min-height: 48px;
        max-height: 48px;
        border: 0px;
		border-image: url(:/graphics/empopup/sliderOff.png);
	    
}

*:checked
{ 
		border-image: url(:/graphics/empopup/sliderOn.png);
}

*:disabled
{ 
		border-image: url(:/graphics/empopup/sliderOffdisabled.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbgraphScrolling">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>133</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>Scrolling</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
